websites/P/Paramount+/presence.ts

```diff
@@ -0,0 +1,205 @@
+const presence = new Presence({
+  clientId: "821433038335377418"
+}), 
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused",
+    live: "presence.activity.live",
+    search: "presence.activity.searching"
+  });
+
+  let title: string,
+    seasonEpi: string,
+    movTitle: string,
+    vidMdTl: Element,
+    elapsed: number = undefined,
+    oldUrl: string = undefined;
+
+  presence.on("UpdateData", async () => {
+
+    const vidArea = document.querySelector(".video__player-area");
+    let video: HTMLVideoElement = null;
+    const presenceData: PresenceData = { 
```

Comment:
move the vidArea down and merge with the const of presenceData

Reply:
Give me a second